File: app/src/main/java/com/neome/java/api/home/main/msg/MsgEntFilterNoVersion.java
// neome.ai API - do not change
//

package com.neome.java.api.home.main.msg;

import com.neome.java.api.meta.base.Types.EntId;
import com.neome.java.api.nucleus.base.msg.Msg;

import org.jetbrains.annotations.Nullable;

import java.util.Set;

@SuppressWarnings("unused")
public class MsgEntFilterNoVersion extends Msg
{
  @Nullable
  public Set<EntId> filterEntIdSet;
}

File: app/src/main/java/com/neome/api/home/main/msg/MsgEntFilterNoVersion.kt
// neome.ai API - do not change
//
@file:Suppress("unused", "MemberVisibilityCanBePrivate", "CanBeParameter", "PropertyName")


package com.neome.api.home.main.msg

import com.neome.api.meta.base.Types.EntId
import com.neome.api.nucleus.base.msg.Msg

interface MsgEntFilterNoVersion : Msg
{
  val filterEntIdSet: Set<EntId>?
}